packages/block-library/src/paragraph/index.js

```diff
@@ -201,20 +200,15 @@ export const settings = {
 			migrate( attributes ) {
 				return {
 					...attributes,
-					content: [
-						<RawHTML key="html">{ attributes.content }</RawHTML>,
-					],
+					content: richTextStructure.create( attributes.content ),
```

Comment:
The same question applies, can it be DOM indpenendent?

Reply:
This is similar to hpq for sourced block attributes. We're using the browser to parse the HTML. How does that happen on mobile?